backend/src/main/resources/org/cryptomator/hub/flyway/V11__Update_Audit_Events.sql

```diff
@@ -0,0 +1,50 @@
+-- Delete all audit events so that we don't have to deal with migration of old events
+-- This is fine because audit logging hasn't been introduced in a stable release yet
+TRUNCATE "audit_event" CASCADE;
```

Comment:
`TRUNCATE`

This is fine. 🐶🔥

Reply:
I hope it is? 😂 It sounded great!

https://stackoverflow.com/questions/13223820/postgresql-delete-all-content
https://www.postgresql.org/docs/current/sql-truncate.html